package.json

```diff
@@ -28,7 +28,9 @@
         "node",
         "browser"
     ],
-    "dependencies": {},
+    "dependencies": {
+        "globalthis": "^1.0.1"
```

Comment:
why do we need globalthis dep ? `globalThis` or `self` doesnt exist in RN ?

Reply:
Not at all, my bad.
Yes, while `globalThis` does not exist, `self` does.